database/migrations/2023_08_11_172952_create_tests_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('tests', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
Consider adding a unique constraint to the 'test' column if it is intended to store unique values. This will help maintain data integrity by preventing duplicate entries.